code-highlighter.html

```diff
@@ -225,18 +225,21 @@
       _splitHunk: function (e) {
         var path = this._getComposedPath(e)[0];
```

Comment:
I think you missed this in the previous review: this line is equivalent to `var path = e.target`.